app/Nova/Actions/ActivateUitpasIntegration.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Actions;
+
+use App\Domain\Integrations\Models\IntegrationModel;
+use App\Domain\Integrations\Organizer;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Integrations\Repositories\OrganizerRepository;
+use App\Domain\Organizations\Models\OrganizationModel;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Collection;
+use Laravel\Nova\Actions\Action;
+use Laravel\Nova\Actions\ActionResponse;
+use Laravel\Nova\Fields\ActionFields;
+use Laravel\Nova\Fields\Select;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Ramsey\Uuid\Uuid;
+
+final class ActivateUitpasIntegration extends Action
+{
+    use InteractsWithQueue;
+    use Queueable;
+
+    public function __construct(
+        private readonly IntegrationRepository $integrationRepository,
+        private readonly OrganizerRepository $organizerRepository
+    ) {
+    }
+
+    public function handle(ActionFields $fields, Collection $integrations): ActionResponse
+    {
+        /** @var IntegrationModel $integration */
+        $integration = $integrations->first();
+
+        /** @var string $organizationIdAsString */
+        $organizationIdAsString = $fields->get('organization');
+        $organizationId = Uuid::fromString($organizationIdAsString);
```

Comment:
make   $organizationIdAsString inline, and you can drop the docblock

Reply:
Maybe best not to do this is PHPStorm complains about it, it's the same in other Action(s). So maybe just leave it as it is?